mods/cnc/maps/nod07c/rules.yaml

```diff
@@ -0,0 +1,139 @@
+World:
+	LuaScript:
+		Scripts: nod07c.lua
+	MusicPlaylist:
+		StartingMusic: justdoit
+		VictoryMusic: nod_win1
+	MissionData:
+		Briefing: GDI has left themselves open to the deadliest weapon of all, information.\n\nOur spies tell us the GDI base in your area contains an advanced attack helicopter, the Orca.\n\nSteal this weapon, then fly to the nearby village and destroy it.\n\nThe attack will be blamed on GDI.
+		BriefingVideo: nod7b.vqa
+		StartVideo: bcanyon.vqa
+		WinVideo: x.vqa
+		LossVideo: flag.vqa
+	SmudgeLayer@CRATER:
+		InitialSmudges:
+			cr1 40,22 2
+
+Player:
+	PlayerResources:
+		DefaultCash: 0
+
+^TechBuilding:
+	Capturable:
+		Type: ~disabled
```

Comment:
So that new players don't end up wasting 3 of their 4 engineers at the oil derricks, as they are restricted to capturing the hpad only anyway.